index.js

```diff
@@ -8,8 +8,51 @@ const http = require('http'),
         The time is: ${rightNow}
       </p>
     </body>
+ 
+      <body>
+   <div style="position:  absolute; bottom: 5px; background-color: white">
+<h2> Characters in The Lord of The Rings </h2>
+ <ul>
```

Comment:
Might want to check the requirements again... looking for an ordered list here `<ol>` ;-) That's the only thing in the list that needs to change to match the requirements.